lib/galaxy/dependencies/__init__.py

```diff
@@ -300,6 +300,12 @@ def check_hvac(self):
     def check_pkce(self):
         return self.pkce_support
 
+    def check_rucio_clients(self):
+        return sys.version_info >= (3, 19)
+
+
+#        return "rucio" in self.object_stores
```

Comment:
```suggestion
        return sys.version_info >= (3, 9) and "rucio" in self.object_stores
```

?

Reply:
Though maybe skip the version_info check, if the dependency is not available we get a better error

Reply:
If we skip version_info check, the GitHub integration tests will be failing, unless there is a better way to disable them for Python <3.9

Reply:
You can check what python version you're in `setUpClass` of BaseRucioObjectStoreIntegrationTestCase and raise `unittest.SkipTest("rucio requires python >=3.9")`